great_expectations/expectations/core/schemas/ExpectTableRowCountToEqual.json

```diff
@@ -0,0 +1,134 @@
+{
+    "title": "ExpectTableRowCountToEqual",
```

Comment:
I see you called `schema_json()` to generate this and I'm surprised this title doesn't say `Expect Table Row Count To Equal` ☹️

Reply:
I would suggest customizing that by making a base Expectation class separate (add space) on case changes for `title`

Reply:
@Kilo59 we already added the logic. That's why I was surprised I didn't see it.

Reply:
@dctalbot @NathanFarmer 
I'm just saying you'd need to customize how the title is rendered (which you could at the base class (but you'd have to be careful not to override it.
https://docs.pydantic.dev/1.10/usage/schema/#schema-customization

Something like this...
```python
class MyBaseClass(pydantic.BaseModel):
  foo: Literal['bar']
  
  class Config:
      @staticmethod
      def schema_extra(schema: Dict[str, Any], model: Type['MyBaseClass']) -> None:
          schema["title"] = camel_case_to_words_with_spaces(schema["title"])
```

Reply:
or...

```python
class MyBaseClass(pydantic.BaseModel):
  foo: Literal['bar']

  @classmethod
  def _title_transform(cls) -> str:
    title = cls.schema()["title"]
    return camel_case_to_words_with_spaces(title)

  class Config:
      @staticmethod
      def schema_extra(schema: Dict[str, Any], model: Type['MyBaseClass']) -> None:
          schema["title"] = model._title_transform()


class ChildA(MyBaseClass):
  pass

class ChildB(MyBaseClass):

  class Config:
      @staticmethod
      def schema_extra(schema: Dict[str, Any], model: Type['ChildB']) -> None:
          other_transforms(schema)
          schema["title"] = model._title_transform()

print(MyBaseClass.schema()["title"]
print(ChildA.schema()["title"])
print(ChildB.schema()["title"])
```
```
My Base Class
Child A
Child B
```